source/LoadPanel.cpp

```diff
@@ -416,6 +416,10 @@ void LoadPanel::UpdateLists()
 	for(const string &path : fileList)
 	{
 		string fileName = Files::Name(path);
+		// Skip the Steam Auto Cloud file, which is created by Steam but not a real save file.
+		if(fileName == "steam_autocloud.vdf")
+			continue;
+
 		// The file name is either "Pilot Name.txt" or "Pilot Name~SnapshotTitle.txt".
```

Comment:
This comment already assumes that we're looking at a text file, so why not just exclude everything that isn't a text file?